drracket/scribblings/drracket/menus.scrbl

```diff
@@ -543,3 +543,47 @@ brings the corresponding window to the front.
    DrRacket to other languages.}
 
 ]
+
+
+@; ----------------------------------------
+
+@section{@onscreen{Context menu}}
+
+The following additional menu items may be available when activating the context menu in an 
+editing panel. 
```

Comment:
We don't use the phrase "editing panel" either. For this one, I think that "either the definitions or interactions window" is probably the right phrase.

Reply:
yes you do! 

> 1 Interface Essentials
> 
> The DrRacket window has three parts: a row of buttons at the top, **two editing panels** in the middle, and a status line at the bottom.

<img width="784" alt="image" src="https://github.com/user-attachments/assets/64ba2506-06be-4dbf-8efd-325efb3786a9">